google-cloud-bigquery/acceptance/bigquery/location_test.rb

```diff
@@ -58,9 +58,7 @@
     insert_response.error_rows.must_be :empty?
 
     job_id = "test_job_#{SecureRandom.urlsafe_base64(21)}" # client-generated
-    query_job = dataset.query_job query, job_id: job_id do |j|
-      j.location = region
```

Comment:
Why was `location` removed from the tests in this file?

Reply:
Because now that the location acquired from the dataset or table, this configuration is no longer needed.

Reply:
Makes sense.